app/src/main/java/org/mozilla/focus/utils/UrlUtils.kt

```diff
@@ -154,6 +155,9 @@ object UrlUtils {
 
     @JvmStatic
     fun toDisplayUrl(url: String): String {
-        return ""
+        return when (url) {
+            BrowserFragment.APP_URL_HOME -> ""
```

Comment:
I think we could do better than using the empty string to represent the intent of "use the hint text". e.g. `Either<String, HintText>`?

Given that we're pressed for time, we can leave this as is for now and at least document:
- Document in the function javadoc what the return values are
- Document inline why the empty string (because it forces TextView to show the intent)